Cargo.toml

```diff
@@ -1,17 +1,20 @@
 [package]
-name = "organisationsnummer"
-description = "Validate Swedish organization numbers"
-version = "1.0.0"
 authors = ["Fredrik Forsmo <fredrik.forsmo@gmail.com>"]
+description = "Validate Swedish organization numbers"
 edition = "2021"
+homepage = "https://organisationsnummer.dev"
 license = "MIT"
+name = "organisationsnummer"
 readme = "README.md"
-homepage = "https://organisationsnummer.dev"
 repository = "https://github.com/organisationsnummer/rust"
+version = "1.0.0"
```

Comment:
Is this automatically updated? I expect so, but just want to doublecheck so we don't replace a package already in cargo :P

Reply:
Ah, no. I'll fix it!